### PR TITLE
[hue] Changed default color mode for color commands to XY

### LINKED ISSUE #10159: [hue] [zigbee] openhab does not change color on a bulb or zone, if the bulb or zone is in white mode
OpenHAB 2.5.12

When changing the color on a zone, room or bulb in OpenHAB using the PaperUI or similar, and if the color bulb is in warm white/white mode, the change of color is ignored or in some cases a random white color is selected. 

This behaviour was not the case in early versions of 2.5. It's been a while since I used/tested it.

I'm not sure, if #9741 refers to the same issue or a similar cause, but I feel, that when one sets a color or a warm white tone in OpenHAB, OpenHAB should be able to switch the bulb to the correct mode first, if needed.

If I use the Hue App (or equivalent) and set the bulb, room or zone to a color first, then OpenHAB will correctly set colors.

/M

### PR DESCRIPTION
**Pull request by cweitkamp**
- Changed default color mode for color commands to XY

Fixes #10159 

Signed-off-by: Christoph Weitkamp <github@christophweitkamp.de>

### CODE REVIEW

File: bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/handler/LightStateConverter.java [hunk 1]
```diff
@@ -68,8 +68,10 @@
      * @return light state representing the {@link HSBType}.
      */
     public static StateUpdate toColorLightState(HSBType hsbType, State lightState) {
-        StateUpdate stateUpdate = ColorMode.XY.equals(lightState.getColorMode()) ? toXYColorLightState(hsbType)
-                : toHSBColorLightState(hsbType);
+        // XY color more is the implicit default: Use XY color mode if i) no color mode is set or ii) if the bulb is in
```
Comment (lolodomo) [inline bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/handler/LightStateConverter.java:71]: is more

**Review by lolodomo — CHANGES_REQUESTED** [review]
Just an ultra minor typo in a comment.
Was this change validated with official hue color lights ?

File: bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/handler/LightStateConverter.java (hunk 1 above)
Comment (cweitkamp) [reply bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/handler/LightStateConverter.java]: fixed

**Comment by cweitkamp** [comment]
>  Was this change validated with official hue color lights ?

Yes. I am running this for couple of days in my environment and it is working nice. You can find some links to Hue API docs to clarify on the change in https://github.com/openhab/openhab-addons/issues/10159#issuecomment-814787490.

**Review by lolodomo — APPROVED** [review]

**Comment by lolodomo** [comment]
I am confident on you ;) I will let you know if I encounter problem once it is merged.

**Review by fwolter — APPROVED** [review]

**Pull request merged**